packages/realm_dart/lib/src/cli/atlas_apps/baas_client.dart

```diff
@@ -29,14 +29,46 @@ class BaasAuthHelper {
   }
 }
 
+enum AppNames {
```

Comment:
```suggestion
enum AppName {
```
I don't think we should use plural on `enums`. It looks odd in argument lists and on call-sites.